migration/v0.44/README.md

```diff
@@ -124,6 +124,13 @@ As with all scripts in this repository `CK8S_CONFIG_PATH` is expected to be set.
     ./bin/ck8s update-ips ${CK8S_CLUSTER} apply
     ```
 
+1. Remove falco-exporter config and uninstall it:
+
+    ```bash
+    ./migration/v0.44/prepare/10-remove-falco-exporter-config.sh
+    ./migration/v0.44/apply/20-uninstall-falco-exporter.sh execute
```

Comment:
We usually do all changes in the env under the `### Apply upgrade - _disruptive_` section. should you move this apply step to that part as well?

Reply:
I'll move it.

Reply:
I moved it to the first step in the apply upgrade-disruptive.

Reply:
Or have both in just the second section

Reply:
Moved all to second section.